cmplibrary/src/main/java/com/sourcepoint/cmplibrary/core/web/ConsentWebView.kt

```diff
@@ -190,6 +196,18 @@ internal class ConsentWebView(
 
         @JavascriptInterface
         override fun onAction(actionData: String) {
+
+            if (context.isInternetConnected().not()) {
+                jsClientLib.onError(
+                    view = this@ConsentWebView,
+                    error = NoInternetConnectionException(
+                        description = context.getString(R.string.exception_no_internet_connection_text)
+                    )
+                )
+                jsClientLib.dismiss(this@ConsentWebView)
+                return
+            }
+
```

Comment:
@bohdan-go-wombat if the network requests are checking for connectivity, I believe we don't need to check on `onAction`.

Reply:
@andresilveirah this is the case when the user had connection when getting a First Layer Message, but lost it before taking an action, so for the sake of additional sanity check we should leave it as is

Reply:
@andresilveirah deleted